### PR TITLE
feat(provider/ecs): Added EnableAsg stage

### PR DESCRIPTION
**Pull request by BrunoCarrier**
feat(provider/ecs): Added EnableAsg stage

@anotherchrisberry @icfantv @jrsquared @christopherthielen

### CODE REVIEW

File: app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.enableAsgStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'enableServerGroup',
+      alias: 'enableAsg',
+      cloudProvider: 'ecs',
+      templateUrl: require('./enableAsgStage.html'),
+      executionDetailsUrl: PipelineTemplates.enableAsgExecutionDetails,
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js]: PipelineTemplates.enableAsgExecutionDetails no longer exists, you should be able to just omit the executionDetailsUrl property and the default execution details sections will be picked up from the default destroy asg stage.

File: app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const angular = require('angular');
+
+import { PipelineTemplates, StageConstants } from '@spinnaker/core';
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js]: Don't need to import `PipelineTemplates` anymore.

File: app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js (hunk 1 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js]: Done

File: app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js (hunk 2 above)
Comment (BrunoCarrier) [reply app/scripts/modules/ecs/pipeline/stages/enableAsg/ecsEnableAsgStage.js]: Done

**Pull request merged**